libraries/c_sdk/aws/defender/src/private/aws_iot_defender_internal.h

```diff
@@ -236,20 +249,11 @@
  * Define encoder/decoder based on configuration AWS_IOT_DEFENDER_FORMAT.
  */
 #if AWS_IOT_DEFENDER_FORMAT == AWS_IOT_DEFENDER_FORMAT_CBOR
-
-    #define DEFENDER_FORMAT     "cbor"
-    #define _defenderEncoder    _IotSerializerCborEncoder       /**< Global defined in iot_serializer.h . */
-    #define _defenderDecoder    _IotSerializerCborDecoder       /**< Global defined in iot_serializer.h . */
-
+    #define DEFENDER_FORMAT    "cbor"
 #elif AWS_IOT_DEFENDER_FORMAT == AWS_IOT_DEFENDER_FORMAT_JSON
```

Comment:
Minor.. I remember there are SIM issues lingering around for JSON support in Device Defender report. Suggest closing the topic on JSON, and let's keep CBOR only in Defender report. We are very memory constrained...

(Keeping "JSON" in code is hinting subtly that we are going to support it. But really, no... )

Reply:
OK will remove.